api/integrations/launch_darkly/services.py

```diff
@@ -109,15 +120,513 @@ def _create_tags_from_ld(
+
+    :param clauses: a list of clauses from Launch Darkly's targeting rule.
+    :param feature: the feature to target for identities.
+    :param segments_by_ld_key: a mapping from Launch Darkly segment key to Segment. Used for "segmentMatch" op.
+    :param rule_name: the name of the rule this feature-specific segment is created for.
+    :return: a list of FeatureState objects for the newly created feature-specific segments.
+    """
+    # There is no "segmentMatch" operator in flagsmith, instead we create a targeting rule for that
+    # specific segment.
+    if "segmentMatch" in [clause["op"] for clause in clauses]:
+        return _create_feature_segments_for_segment_match_clauses(
+            import_request=import_request,
+            clauses=clauses,
+            project=project,
+            feature=feature,
+            environment=environment,
+            segments_by_ld_key=segments_by_ld_key,
+        )
+
+    # Create a feature specific segment for the rule.
+    segment, _ = Segment.objects.update_or_create(
+        name=rule_name, project=project, feature=feature
+    )
+
+    # Create a targeting rule for the new feature-specific segment.
+    _create_segment_rule_for_segment(
+        import_request=import_request,
+        segment=segment,
+        clauses=clauses,
+    )
+
+    # Tie the feature and segment together.
+    feature_segment, _ = FeatureSegment.objects.update_or_create(
+        feature=feature,
+        segment=segment,
+        environment=environment,
+    )
+
+    # Enable rules by default. In LD, rules are enabled if the flag is on.
+    return [
+        FeatureState.objects.update_or_create(
+            feature=feature,
+            feature_segment=feature_segment,
+            environment=environment,
+            defaults={"enabled": True},
+        )[0]
+    ]
+
+
+def _import_targets(
+    import_request: LaunchDarklyImportRequest,
+    ld_flag_config: ld_types.FeatureFlagConfig,
+    feature: Feature,
+    environment: Environment,
+    segments_by_ld_key: dict[str, Segment],
+    mv_feature_options_by_variation: dict[str, MultivariateFeatureOption],
+) -> None:
+    """
+    Import the individual targeting rules for the given Launch Darkly's feature flag.
+
+    :param ld_flag_config: the feature flag config from Launch Darkly.
+    :param feature: the feature to target for identities.
+    :param environment: the environment to target for identities.
+    :param mv_feature_options_by_variation: a mapping from variation index to MultivariateFeatureOption if the
+    flag is multivariate.
+    """
+    if "targets" in ld_flag_config:
+        # Identifiers are grouped by their variation index. So each target has the same variation index.
+        for target in ld_flag_config["targets"]:
+            # Create a segment override for those identities. This is a work-around to support individual
+            # targeting in local evaluation mode.
```

Comment:
Please add a TODO to remove this when local evaluation is supported for identity overrides.

Reply:
Added a link that tracks SDK support for identity overrides.